mmv1/templates/terraform/examples/private_service_connect_google_apis_no_automate_dns.tf.erb

```diff
@@ -0,0 +1,40 @@
+// START vpc_subnet_private_access
```

Comment:
Could you please remove these comments from the example file? From looking at the file history I think they were copied in from elsewhere, where there are comments like `# [START vpc_subnet_private_access]`. Comments like that are used in automation and rely on using unique names, so having these comments here could be confusing to others.

Reply:
Hi @SarahFrench, great thanks, didn't know about the usage of those comments in the automation process, makes sense removing that.